### DISCUSSION TITLE
Software License for TBB

### DISCUSSION THREAD

**Discussion by haowu74** [oneapi-src/oneTBB#1554] [Design discussions]
We are using TBB 4.4 Update 4 in one of our 3rd party component. The Black Duck scan raises a license risk on it.

What is the license for TBB 4.4 Update 4? According to Readme.md it is under GPLv2, however according to [End User License Agreements](https://www.intel.com/content/www/us/en/developer/articles/license/end-user-license-agreement.html), oneAPI TBB is under Apache 2.0.

![image](https://github.com/user-attachments/assets/ead8b4b0-8bba-42ed-94c1-145a34002e50)

I understand the latest versions of TBB are under Apache 2.0, but how about the earlier ones such as TBB 4.4 Update 4? Does the Apache 2.0 replace all the former ones?

**Comment by omalyshe** [comment]
@haowu74 TBB 4.4. Update 4 was released under [GPLv2](https://github.com/oneapi-src/oneTBB/blob/tbb_4.4/COPYING) with the runtime exception. Although the latest TBB version - oneTBB 2022.0.0 - is released under Apache 2.0 it does **not** automatically replaces the license for previous versions.
